rust/merkledb/src/async_chunk_iterator.rs

```diff
@@ -402,6 +399,24 @@ lazy_static! {
     };
 }
 
+// Annoying that we have to explicitly declare this, but that is the cost of using async_trait
```

Comment:
Change to the reason being the use of Pin with a struct containing unsafe pointers.